### ISSUE TITLE
Make website more WCAG compliant

### ISSUE THREAD

**Issue by Fulgar** [Fulgar/PortfolioWebsite#18]
Website could be way more WCAG compliant. Aria-labels, proper alt text, color contrast, etc. should be reviewed and revised to be more WCAG compliant.
